spec/view_objects/project_documentation_view_object_spec.rb

```diff
@@ -98,6 +98,8 @@
     let(:project) { double('Project', id: 10, user: build(:user)) }
     let(:project_documentation) { described_class.new(project: project, banks: []) }
 
+    before { allow(project).to receive(:decorate).and_return(project) }
```

Comment:
@carloslopes I have created another spec for this case, please, check the spec again. And thanks for your suggestion. 😃